app/code/Magento/UrlRewrite/Block/Edit.php

```diff
@@ -65,7 +65,7 @@ public function __construct(
      */
     protected function _prepareLayout()
     {
-        $this->setTemplate('edit.phtml');
+        $this->setTemplate('Magento_UrlRewrite::edit.phtml');
```

Comment:
Please, revert fix in places when the template specified dynamically

Reply:
Done.